docker-compose.yml

```diff
@@ -9,7 +9,7 @@ services:
         - http_proxy
         - https_proxy
         - no_proxy
-    image: hmcts.azurecr.io/hmcts/cmc-pdf-service
+    image: hmctspublic.azurecr.io/cmc/pdf-service
```

Comment:
perhaps one day it can be owned by platform. just saying

Reply:
once it goes live on AKS we plan on changing it